hybrid/flow.py

```diff
@@ -258,10 +259,10 @@ def halt(self):
 
 
 class ParallelBranches(Runnable, traits.SIMO):
-    """Runs multiple :class:`Branch`es in parallel, blocking until all finish.
+    """Runs multiple :class:`Branch` classes in parallel, blocking until all finish.
```

Comment:
Again, it's more general than `Branch` - it's `Runnable`, or workflow.

Reply:
as above